### PR TITLE
More robust version checking of underlying phpUnit

### PR DESCRIPTION
**Pull request by chandeeland**
... the changes are not caught by the function_exists test, so i changed to to check PHPUnit's version function

### CODE REVIEW

File: classes/kohana/unittest/tests.php [hunk 1]
```diff
@@ -51,7 +51,8 @@ static public function configure_environment($do_whitelist = TRUE, $do_blacklist
 		spl_autoload_register(array('Unittest_tests', 'autoload'));
 
 		// As of PHPUnit v3.5 there are slight differences in the way files are black|whitelisted
-		self::$phpunit_v35 = function_exists('phpunit_autoload');
+                list($major, $minor, $release) = explode('.', PHPUnit_Runner_Version::id());
```
Comment (hkdobrev) [inline classes/kohana/unittest/tests.php:54]: Why not using [`version_compare`](http://php.net/manual/en/function.version-compare.php)?

**Comment by zeelot** [comment]
This pull doesn't seem mergable any more but like @hkdobrev said, `version_compare` should be simpler. You also seem to be using `&&` and `||` instead of `AND` and `OR`.

I would also like to simply get rid of any version comparisons from now on anyway since we include phpunit using composer so we only ever have to support the version we are requiring in `composer.json`.

**Comment by hkdobrev** [comment]
> I would also like to simply get rid of any version comparisons from now on anyway since we include phpunit using composer so we only ever have to support the version we are requiring in composer.json.

:+1: on what @Zeelot said. This is one of the best sides of using Composer. Support only what you use and not everything available.

**Comment by zombor** [comment]
I'm pretty sure I got rid of all that. Maybe it was in the 3.3 branch.

**Comment by zeelot** [comment]
You're right, all this sillyness is gone from 3.3 :) excellent.

**Pull request closed without merging**